hack/deploy_loki_using_observatorium_yaml.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+LOKI_PROJECT_NAME=observatorium-logs-test
+LOKI_S3_SECRET_NAME=loki-objectstorage-secret
+AWS_CREDENTIALS_FILE_NAME=~/.aws/credentials
+
+LOKI_INGESTER_REPLICAS=${LOKI_INGESTER_REPLICAS:-"3"}
+
+get_aws_credentials() {
+  # get AWS credentials from local user account file
+  AWS_S3_USER_ACCESS_KEY=$(grep -m 1 aws_access_key_id < $AWS_CREDENTIALS_FILE_NAME | awk '{print $3}')
+  AWS_S3_USER_SECRET_KEY=$(grep -m 1 aws_secret_access_key < $AWS_CREDENTIALS_FILE_NAME | awk '{print $3}')
+
+  # Get AWS credentials from environment variables
+  AWS_S3_END_POINT="${AWS_S3_END_POINT:-https://s3.us-east-1.amazonaws.com}"
+  AWS_S3_REGION="${AWS_S3_REGION:-us-east-1}"
+  AWS_S3_LOKI_BUCKET_NAME="${AWS_S3_LOKI_BUCKET_NAME:-loki-bucket}"
+  AWS_S3_ACCESS_KEY="${AWS_S3_ACCESS_KEY:-$AWS_S3_USER_ACCESS_KEY}"
+  AWS_S3_SECRET_KEY="${AWS_S3_SECRET_KEY:-$AWS_S3_USER_SECRET_KEY}"
```

Comment:
nit. for easy recognition by a novice user let's keep env var named as the original fields, e.g.:
```
AWS_USER_ACCESS_KEY_ID -> aws_access_key_id
AWS_SECRET_ACCESS_KEY -> aws_secret_access_key
```

Reply:
done